src/avizo_client.vala

```diff
@@ -91,9 +91,16 @@ public class AvizoClient : GLib.Application
 		Object(application_id: "org.danb.avizo.client",
 		       flags: ApplicationFlags.HANDLES_COMMAND_LINE);
 
-		_service = Bus.get_proxy_sync(BusType.SESSION,
-		                              "org.danb.avizo.service",
-		                              "/org/danb/avizo/service");
+		try
+		{
+			_service = Bus.get_proxy_sync(BusType.SESSION,
+			                              "org.danb.avizo.service",
+			                              "/org/danb/avizo/service");
+		}
+		catch (IOError e)
+		{
+			stderr.printf(@"avizo: $(e.message)\n");
+		}
```

Comment:
If I understand it correctly, this will print the message and discard the exception, so it will continue with `_service` being null and blown up later (probably on undefined behaviour?).